README.md

```diff
@@ -1,306 +1,61 @@
-- `CERT_ROTATE_INTERVAL`:  Expiration time for server certificates, default is half a year
-
-## Develop
-
-This project uses [kubevirtci](https://github.com/kubevirt/kubevirtci) to
-deploy local cluster.
+  template:
+    spec:
+      domain:
+        devices:
+          interfaces:
+          - macAddress: "02:00:00:00:00:00"
+            masquerade: {}
+            name: default
+...
+      networks:
+      - name: default
+        pod: {}
+...
+```
+
+For more Examples see [VirutalMachine Allocation Example Guide](doc/vm-creation-example.md) and [Pod Allocation Example Guide](doc/pod-creation-example.md)
 
-### Dockerized Kubernetes Provider
-
-Refer to the [kubernetes 1.17](https://github.com/kubevirt/kubevirtci/blob/master/cluster-up/cluster/k8s-1.17/README.md)
-
-### Usage
-
-Use following commands to control it.
-
-*note:* Default Provider is one master + one worker of kubernetes 1.17 nodes
-with multus cni plugin.
-
-#### Ephemeral cluster
-
-```shell
-# Deploy local Kubernetes cluster
-export KUBEVIRT_NUM_NODES=3 # one master + two workers
-make cluster-up
-
-# SSH to node01 and open interactive shell
-./cluster/cli.sh ssh node01
-
-# SSH to node01 and run command
-./cluster/cli.sh ssh node01 echo 'Hello World'
-
-# Communicate with the Kubernetes cluster using kubectl
-./cluster/kubectl.sh
-
-# Build project, build images, push them to cluster's registry and install them
-make cluster-sync
-
-# Destroy the cluster
-make cluster-down
-
-# Run function test
-make functest
-```
-#### External cluster
-
-To deploy at an external cluster not provided by kubevirtci we can to do
-the following steps
-
-```bash
-# Set env variables
-export KUBEVIRT_PROVIDER=external
-export KUBECONFIG=[path to the kubeconfig of the cluster]
-
-# Intermediate registry to deploy kubemacpool
-export DEV_REGISTRY=docker.io # can be different
-
-# Intermediate repo to deploy kubemacpool
-export REPO=$USER # Usually there is a docker.io/$USER this can use
-
-# Deploy kubevirt and multus
-make cluster-up
+### kubemacpool service
 
-# Build project, build images, push them to $DEV_REGISTRY/$REPO/kubemacpool and
-# install it
-make cluster-sync
+The user can specify whether MAC Address will be allocated to all VirtualMachines and Pods by default or not.
+For More information, see [Kubemacpool Opt-Modes](doc/opt-modes.md)
 
-# Destroy the cluster, it will not install kubevirt or multus
-make cluster-down
+### webhook certificate handling
 
-# Run function test
-make functest
-```
+Part of the kubemacpool functionality is implemented as a [kubernetes mutating admission webhook](https://kubernetes.io/doc/reference/access-authn-authz/admission-controllers/#mutatingadmissionwebhook) 
+to ensure that the MAC Address is assigned before VirtualMachine is created.
+This webhook uses self signed certificate. For more information see [Certificate handling](doc/certificate-handling.md)
```

Comment:
Is there a reason to have Certificate with capital?
I'm not sure but I don't think so

```suggestion
This webhook uses self signed certificate. For more information see [certificate handling](doc/certificate-handling.md)
```

Reply:
I don't think so.. changed. Done